src/QiskitCompletionProvider.ts

```diff
@@ -131,16 +135,25 @@ export class QiskitInlineCompletionProvider
   readonly name: string = 'Qiskit Code Assistant';
 
   app: JupyterFrontEnd;
+  settings: ISettingRegistry.ISettings;
   prompt_id: string = '';
-  schema: ISettingRegistry.IProperty = {
-    default: {
-      enabled: true,
-      timeout: 10000
```

Comment:
from what I can tell we can achieve the same results by just increasing this value instead of the surrounding code changes